sound/soc/sof/debug-dsp-ops.c

```diff
@@ -252,5 +284,9 @@ int sof_dbg_dsp_ops_test_init(struct snd_sof_dev *sdev)
 	if (ret < 0)
 		return ret;
 
-	return sof_dsp_dsp_ops_create_dfse(sdev, "dsp_power_state", dsp_ops_debugfs, 0666);
+	ret = sof_dsp_dsp_ops_create_dfse(sdev, "dsp_power_state", dsp_ops_debugfs, 0666);
+	if (ret < 0)
+		return ret;
+
+	return sof_dsp_dsp_ops_create_dfse(sdev, "fw_state", dsp_ops_debugfs, 0444);
```

Comment:
fwiw, we are exposing the `fw_state` as a read-only buffer.
Which is not too user friendly, I agree.

Reply:
making it read-write would be too confusing with all the stages we have for fw boot isnt it?

Reply:
No, what I mean is that we have this information already, but it is just returning the raw u32.
I don't know if it is used by anyone.